src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  onCloseModal: () => void;
+  todo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = props => {
+  const { onCloseModal, todo } = props;
+
+  const [loading, setLoading] = useState(true);
```

Comment:
the same issue(fix it for all such cases)
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```